scripts/polarion-test-run-upload.sh

```diff
@@ -0,0 +1,27 @@
+# Populate token-prefix and betelgeuse depending on Satellite version.
+
+TOKEN_PREFIX=""
+POLARION_SELECTOR="name=Satellite 6"
+SANITIZED_ITERATION_ID="${TEST_RUN_ID//[. ]/_}"
+TEST_RUN_GROUP_ID="$(echo ${TEST_RUN_ID} | cut -d' ' -f2)"
+
+# Prepare the XML files
+
+betelgeuse ${TOKEN_PREFIX} test-run \
+    --custom-fields "isautomated=true" \
+    --custom-fields "arch=x8664" \
+    --custom-fields "variant=server" \
+    --custom-fields "plannedin=${SANITIZED_ITERATION_ID}" \
+    --response-property "${POLARION_SELECTOR}" \
+    --test-run-title "${TEST_RUN_ID}" \
+    --test-run-id "${TEST_RUN_ID}" \
+    --test-run-group-id "${TEST_RUN_GROUP_ID}" \
+    --status inprogress \
+    "./${ENDPOINT}-results.xml" \
+    tests/foreman \
+    "${POLARION_USERNAME}" \
+    "${POLARION_PROJECT}" \
+    "sat-${IMPORTANCE}-results.xml"
```

Comment:
I think this (and the ENDPOINT-results.xml on L20) needs to be more configurable. Its relevant to how we're writing CI, but these envvars won't be relevant or set for component owners trying to upload results.

Reply:
oops I think I already amended the line, watch me repushing PR